distribution/lib/Standard/Table/0.0.0-dev/src/Error.enso

```diff
@@ -55,6 +55,16 @@ Duplicate_Column_Selectors.to_display_text : Text
 Duplicate_Column_Selectors.to_display_text =
     "The provided Column_Selector has duplicate entries: "+this.duplicate_selectors.short_display_text+"."
 
+## Indicates that one column has been matched by multiple selectors.
+
+   In case the selectors have differing metadata and the error does not prevent
+   the operation from continuing, the first selector on the list is used.
+type Column_Matched_By_Multiple_Selectors (column_name : Text) (selectors : [Any])
```

Comment:
We should be consistent on `[Any]` versus `Vector Any`

Reply:
As discussed, we'll make a separate task to align these, as this applies to the whole codebase.